File: alpaul_gls/Content/Downloadable/PHAO2018092520181018161350/Business/AspNetUserClaim.vb
Imports System.Collections.Generic

Public Class AspNetUserClaim
    Public Shared Function Add(value As Models.AspNetUserClaim) As Integer
        Return Data.AspNetUserClaim.Add(value)
    End Function

    Public Shared Function Edit(value As Models.AspNetUserClaim) As Integer
        Return Data.AspNetUserClaim.Edit(value)
    End Function

    Public Shared Function Delete(value As String) As Integer
        Return Data.AspNetUserClaim.Delete(value)
    End Function

    Public Shared Function GetById(value As String) As Models.AspNetUserClaim
        Return Data.AspNetUserClaim.GetById(value)
    End Function

    Public Shared Function GetByGUID(value As String) As Models.AspNetUserClaim
        Return Data.AspNetUserClaim.GetByGUID(value)
    End Function

    Public Shared Function GetAll() As List(Of Models.AspNetUserClaim)
        Return Data.AspNetUserClaim.GetAll()
    End Function
End Class

File: alpaul_gls/Content/Downloadable/PHAO2018092520181018171036/Business/AspNetUserClaim.cs
using System.Collections.Generic;

namespace Business
{
    public class AspNetUserClaim
    {
	    private static Data.AspNetUserClaim obj = new Data.AspNetUserClaim();

        public static int Add(Models.AspNetUserClaim value)
        {
            return obj.Add(value);
        }

        public static int Edit(Models.AspNetUserClaim value)
        {
            return obj.Edit(value);
        }

        public static int Delete(stirng value)
        {
            return obj.Delete(value);
        }

        public static Models.AspNetUserClaim GetById(string ID)
        {
            return obj.GetById(ID);
        }

		public static Models.AspNetUserClaim GetByGUID(string ID)
        {
            return obj.GetByGUID(ID);
        }

        public static List<Models.AspNetUserClaim> GetAll()
        {
            return obj.GetAll();
        }
    }
}